notebooks/atlasreader.ipynb

```diff
@@ -50,7 +50,7 @@
     "\n",
     "If you use `atlasreader` in your publication, please cite the following paper:\n",
     "\n",
-    "    Markello R., Gale D., Herholz P., Whitaker K., & Notter M. P. (2019). AtlasReader: Quick generation of coordinate tables and region labels for statistical MRI images. The Journal of Open Source Software, ZZZ, YYY.\n",
+    "    Markello R., Gale D., Herholz P., Whitaker K., Notter-Bielser M.-L., & Notter M. P. (2019). AtlasReader: Quick generation of coordinate tables and region labels for statistical MRI images. The Journal of Open Source Software, ZZZ, YYY.\n",
```

Comment:
```suggestion
    "    Markello R., Gale D., Herholz P., Whitaker K., Notter-Bielser M.-L., & Notter M. P. (2019). AtlasReader: A Python package to generate coordinate tables, region labels, and informative figures from statistical MRI images. The Journal of Open Source Software, ZZZ, YYY.\n",
```

The title is in so many places!

Reply:
I cannot commit your suggestion, github gives me some errors. Will therefore add it in a new commit again and resolve the conversation. Thank you!